packages/manager/apps/pci-databases-analytics/src/pages/services/create/_components/OrderFunnel.component.tsx

```diff
@@ -143,6 +145,16 @@ const OrderFunnel = ({
 
   const classNameLabel = 'scroll-m-20 text-xl font-semibold';
 
+  const [isApiTerraformDialogOpen, setApiTerraformDialogOpen] = useState(false);
+  const [dialogData, setDialogData] = useState<ReturnType<
+    typeof useOrderFunnel
+  > | null>(null);
+
+  const handleApiTerraformDialog = () => {
+    setDialogData(model);
+    setApiTerraformDialogOpen(true);
+  };
```

Comment:
`dialogData` is not used anywhere
```suggestion
  const handleApiTerraformDialog = () => {
    setApiTerraformDialogOpen(true);
  };
```